lesson_02/job1/tests/test_storage.py

```diff
@@ -0,0 +1,65 @@
+"""
+Tests api.py
+# TODO: write tests
+"""
+import json
+import os.path
+from unittest import TestCase, mock
+from lesson_02.job1.tests.common import remove_dir
+from lesson_02.job1.storage import save_to_disk
+
+class SaveToDiskTestCase(TestCase):
+    temp_dir = '/tmp/de07/tests/raw/sales'
+    test_data = [{
+                "key1": "value1",
+                "key2": "value2",
+                "key3": "value3"
+            },
+            {
+                "key4": "value4",
+                "key5": "value5",
+                "key6": "value6"
+            }]
+    """
+    Test storage.save_to_disk function.
+    """
+    def tearDown(self) -> None:
+        """
+        Here we just delete all the temporary paths etc.
+        """
+        remove_dir(self.temp_dir)
+
+    def read(self, path):
+        data = None
+        with open(path, 'rt+') as f:
+            data = json.load(f)
+        return data
+
+    def testWriteCorrectData(self):
```

Comment:
this is not a pythonic method naming way - pls, read PEP8. 
I told a lot in our lesson about why this is important

Reply:
Exception: setUp and tearDown in a TestCase because those methods are very old